zeppelin-web/src/app/notebook/paragraph/paragraph-control.html

```diff
@@ -26,6 +26,20 @@
   <span class="icon-control-play" style="cursor:pointer;color:#3071A9" tooltip-placement="top" tooltip="Run this paragraph (Shift+Enter)"
         ng-click="runParagraph(getEditorValue())"
         ng-show="paragraph.status!='RUNNING' && paragraph.status!='PENDING' && paragraph.config.enabled"></span>
+  <span ng-show="paragraph.runtimeInfos.jobUrl.length == 1">
+    <a href="{{paragraph.runtimeInfos.jobUrl[0]}}" target="_blank"><span class="fa fa-tasks"></span> Spark job </a>
```

Comment:
And how about we generalize this feature little bit more, such as

```
runtimeInfos = {
   label: "Spark Job"
   jobUrl : [],
}
```

So other interpreters can also leverage this feature.